lib/sort_by_length.rb

```diff
@@ -1,7 +1,31 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(1)
```

Comment:
Technically since you're doing `.split` this makes a new array and the space complexity is O(n)